src/App/Utilities/Extentions.cs

```diff
@@ -55,7 +55,14 @@ public static void FocusWithDelay(this View view, int delay = 1000, bool forceDe
 
         public static async Task PushForDeviceAsync(this INavigation navigation, Page page)
         {
-            await navigation.PushModalAsync(new ExtendedNavigationPage(page), true);
+            if (Device.RuntimePlatform == Device.Windows)
+            {
+                await navigation.PushAsync(page, true);
+            }
+            else
+            {
+                await navigation.PushModalAsync(new ExtendedNavigationPage(page), true);
```

Comment:
Does UWP  not work with modals or something? I thought people were suppose to use the back button, like Android?

Reply:
They do use the back button. It comes up in the desktop version when there is something to go back to (the settings app does this). It only comes up with PushAsync though

Reply:
I think we need a close button then like iOS. There are many pages that use modal that will be affected.

Reply:
Do you mean for other pages that pop up or for this page?

Reply:
Looking at the Xamarin forms documentation this shouldn't be a modal push. Modals are for things that can't be navigated away from (hence the no back button). See here: https://developer.xamarin.com/guides/xamarin-forms/application-fundamentals/navigation/modal/

Would it be better to swap to only using modal sometimes? Either add an extra argument to specify non-modal or create a new function?